src/components/worker/WorkerMasterPanel.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { withTheme, withStyles } from "@material-ui/core/styles";
+import { Paper, Grid, Typography, Divider, Checkbox, FormControlLabel } from "@material-ui/core";
+import {
+  formatMessage,
+  FormattedMessage,
+  PublishedComponent,
+  FormPanel,
+  TextInput,
+  withModulesManager,
+} from "@openimis/fe-core";
```

Comment:
Try to maintain consistency in the import organization. Consider splitting them into three sections: external libraries, UI components and styles, and internal modules and components. Each section should be separated by new line.

Reply:
done